src/client/src/apps/MainRoute/routes/styled.tsx

```diff
@@ -4,7 +4,7 @@ import { rules } from 'rt-styleguide'
 export const OverflowScroll = styled.div`
   overflow-y: scroll;
   ${rules.touchScroll};
-  height: 100vh;
+  height: 100%;
```

Comment:
Have you tested this both in the browser and OpenFin? I think this change was made to make it work in OpenFin.

Reply:
It seems to work in OpenFin as well, though there is a separate issue of the scrollbar not appearing at an early enough width,

![3977ef01a4169d2fe308da89507c2e71](https://user-images.githubusercontent.com/26552273/95876473-187fdc00-0d41-11eb-9059-b014c0bb4272.gif)
/

Reply:
Works on both platforms now:

![7e508e0520d938176d1f2de64af1b69f](https://user-images.githubusercontent.com/26552273/96130242-a84e9300-0ec5-11eb-8e55-8271bd8338e2.gif)